internal/client/client.go

```diff
@@ -196,6 +196,13 @@ func (c *CSAPI) SendEventSynced(t *testing.T, roomID string, e b.Event) string {
 	return eventID
 }
 
+func (c *CSAPI) SendRedaction(t *testing.T, roomID string, e b.Event, eventID string) {
```

Comment:
I think this needs a docstring.

Otherwise seems reasonable to me, though @kegsay might want to check this fits with his view of which helper functions we want in complement.